metadata/990__5__metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "form_990",
+    "retrainable": true,
+    "gpu": 0,
+    "processorType": "CPU",
+    "description": "Machine Learning model (available in Preview) for extracting commonly occurring data points from 990 documents, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key",
+    "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -\u003e Other Services view.",
+    "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+    "changeLog": "",
+    "cpu": 1,
+    "inputType": "JSON",
+    "displayName": "990",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON37_DU",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Document Understanding",
+    "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "22.10",
+    "languageVersion": 4,
+    "version": 5,
+    "customVersion": "22.10.4",
+    "imagePath": "du-semistructured:22.10.4-sf-rc1",
```

Comment:
Just verify the following:
- this image exists in SF repo
- onDemandBundle section has a bundle with this image
- release & onDemand yamls have the same bundle name.
If possible link that PR here